react/Alerter/index.jsx

```diff
@@ -1,142 +1,203 @@
 import React, { Component } from 'react'
-import classNames from 'classnames'
-import { connect } from 'react-redux'
-
-import { translate } from '../I18n'
+import PropTypes from 'prop-types'
+import Portal from 'preact-portal'
 
 import styles from './styles.styl'
+import classNames from 'classnames'
 
-const ALERT_SHOW = 'ALERT_SHOW'
-const ALERT_DISMISS = 'ALERT_DISMISS'
-const ALERT_CLEAR = 'ALERT_CLEAR'
+const createStore = () => {
```

Comment:
Can't we use redux and react-redux? Or something like that https://github.com/developit/state-machine-component?